askup/utils/views.py

```diff
@@ -616,3 +616,28 @@ def create_destroy_subscription(subject_id, user_id, subscribe):
 
         if subscription:
             subscription.delete()
+
+
+def process_organization(organization_id, selected_organization, user_id, url_name, is_rank_list=True):
+    """
+    Process selected organization.
+
+    :return: redirect to a correspondent section if it requires otherwise - None.
+    """
+    if organization_id and selected_organization is None:
+        # Case, when organization_id is specified in the link and restricted to this user
+        return redirect(reverse(url_name, kwargs={'user_id': user_id}))
+
+    if is_rank_list and selected_organization is None:
```

Comment:
does it correct that in `case organization_id and selected_organization is None` we do not care about the `is_rank_list` parameter?

Reply:
that's right. This part of the function code is executed in both cases - in "user profile rank list" view and in "user profile" view.